src/sentry/testutils/cases.py

```diff
@@ -2172,26 +2172,31 @@ def store_functions(
             profile_context["profile_id"] = uuid4().hex
         profile_id = profile_context.get("profile_id")
 
-        timestamp = transaction["timestamp"]
-
         self.store_event(transaction, project_id=project.id)
 
+        timestamp = transaction["timestamp"]
+        trace_context = transaction.setdefault("contexts", {}).setdefault("trace", {})
         functions = [
-            {**function, "fingerprint": self.function_fingerprint(function)}
+            {
+                **function,
+                "self_times_ns": list(map(int, function["self_times_ns"])),
+                "fingerprint": self.function_fingerprint(function),
+            }
             for function in functions
         ]
-
         functions_payload = {
-            "project_id": project.id,
-            "profile_id": profile_id,
-            "transaction_name": transaction["transaction"],
+            "functions": functions,
             # the transaction platform doesn't quite match the
             # profile platform, but should be fine for tests
             "platform": transaction["platform"],
-            "functions": functions,
-            "timestamp": timestamp,
-            # TODO: should reflect the org
+            "profile_id": profile_id,
+            "project_id": project.id,
+            "received": int(datetime.now(tz=timezone.utc).timestamp()),
             "retention_days": 90,
+            "timestamp": int(timestamp),
+            "transaction_name": transaction["transaction"],
+            "transaction_op": trace_context["op"],
+            "transaction_status": trace_context["status"],
```

Comment:
As discussed, these 2 fields don't exist in the dataset, so consider removing them from here and the schema.

Reply:
They do exist in the the dataset: https://github.com/getsentry/snuba/blob/master/snuba/snuba_migrations/functions/0002_add_new_columns_to_raw_functions.py#L27

Reply:
They exist but we don't use them, I'll remove them from here, remove them from the schema, keep them as deprecated fields in the processor until we can remove those columns.